scripts/avocado

```diff
@@ -56,14 +56,6 @@ def handle_exception(*exc_info):
     sys.exit(-1)
 
 
-# simple magic for using scripts within a source tree
-basedir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-if os.path.isdir(os.path.join(basedir, 'avocado')):
-    os.environ['PATH'] += ":" + os.path.join(basedir, 'scripts')
-    os.environ['PATH'] += ":" + os.path.join(basedir, 'libexec')
-    sys.path.append(basedir)
```

Comment:
I'll have to give it a try (after the LTS) but if that works well I'm definitely for this.